frame/support/test/tests/benchmark_ui/missing_origin.stderr

```diff
@@ -0,0 +1,7 @@
+error: Single-item extrinsic calls must specify their origin as the first argument.
+ --> tests/benchmark_ui/missing_origin.rs:5:1
+  |
+5 | #[benchmarks]
+  | ^^^^^^^^^^^^^
```

Comment:
This should point to the function that the `#[extrinsic_call]` is above of.

Reply:
So this one in particular is weird. In the code it is:

```rust
						let Some(origin) = expr_call.args.first().cloned() else {
							return Some(Err(Error::new(expr_call.args.span(), "Single-item extrinsic calls must specify their origin as the first argument.")))
						};

```

But I wonder if something is override that span somewhere?

Reply:
nvm it was apparently because of the clone. If I set the span to a variable then it works fine before the clone